src/main/webapp/webpack.config.js

```diff
@@ -3,6 +3,7 @@ const webpack = require("webpack");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
```

Comment:
Add a file level comment on what this is being used for please.

Reply:
On what this plugin is used for?

Reply:
Just generally on what that particular file is being used for please.  No different than any of of Java files (even in our configuration layer) I'd appreciate a block comment at the top just for a casual viewer to know what the file is being used for.

Reply:
Updated in Updated in d6ef6cd